tests/playwright/sanity/lighthouse.test.ts

```diff
@@ -0,0 +1,27 @@
+import { playAudit } from 'playwright-lighthouse';
+import { test } from '@playwright/test';
+import config from 'lighthouse/lighthouse-core/config/desktop-config';
+import WpAdminPage from '../pages/wp-admin-page';
+import _path from 'path';
+
+test.describe( 'Lighthouse tests', () => {
+	test( 'Accordion widget test', async ( { page }, testInfo ) => {
```

Comment:
If we add more tests here, many/most/all of them will be in this format, we should think about extracting this to a function in the future (not for this PR)

Reply:
yes